packages/adders/paraglide/index.ts

```diff
@@ -61,24 +45,35 @@ export const options = defineAdderOptions({
 	}
 });
 
+const DEFAULT_INLANG_PROJECT = {
+	$schema: 'https://inlang.com/schema/project-settings',
+	modules: [
+		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-empty-pattern@1/dist/index.js',
+		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-identical-pattern@1/dist/index.js',
+		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-missing-translation@1/dist/index.js',
+		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-without-source@1/dist/index.js',
+		'https://cdn.jsdelivr.net/npm/@inlang/message-lint-rule-valid-js-identifier@1/dist/index.js',
+		'https://cdn.jsdelivr.net/npm/@inlang/plugin-message-format@2/dist/index.js',
+		'https://cdn.jsdelivr.net/npm/@inlang/plugin-m-function-matcher@0/dist/index.js'
+	],
+	'plugin.inlang.messageFormat': {
+		pathPattern: './messages/{languageTag}.json'
+	}
+};
+
 export default defineAdder({
 	id: 'paraglide',
 	environments: { svelte: false, kit: true },
 	homepage: 'https://inlang.com',
 	options,
-	packages: [
-		{
-			name: '@inlang/paraglide-sveltekit',
-			version: '^0.11.1',
-			dev: false
-		}
-	],
-	files: [
-		{
-			// create an inlang project if it doesn't exist yet
-			name: () => 'project.inlang/settings.json',
-			condition: ({ cwd }) => !fs.existsSync(path.join(cwd, 'project.inlang/settings.json')),
-			content: ({ options, content }) => {
+	run: ({ sv, cwd, options, typescript, kit, dependencyVersion }) => {
+		const ext = typescript ? 'ts' : 'js';
+		if (!kit) throw new Error('SvelteKit is required');
+
+		sv.dependency('@inlang/paraglide-sveltekit', '^0.11.1');
+
+		if (!fs.existsSync(path.join(cwd, 'project.inlang/settings.json'))) {
+			sv.updateFile('project.inlang/settings.json', (content) => {
```

Comment:
I think we could combine these into a single API that adds the file if it doesn't exist. (maybe `addFile`, `updateFile`, and `upsertFile`? or `updateFile` could take an option describing whether to throw an error or overwrite if an existing file is found?)
```suggestion
		if (!fs.existsSync(path.join(cwd, 'project.inlang/settings.json'))) {
			sv.updateFile('project.inlang/settings.json', (content) => {
```

Reply:
i think a singular API makes sense (similar to how the `name: () => '...'` field operates today where it'll only create the file if the returned string is _not empty_). it could also have a second parameter that indicates the status:
```ts
sv.file('project.inlang/settings.json', (content: string, kind: 'create' | 'update') => {
	if (kind === 'create') {
		// ...
	}
	if (kind === 'update') {
		// ...
	}
});
```
most of the time though, you wouldn't care about whether the file is created or updated, so it can simply be this:
```ts
sv.file('project.inlang/settings.json', (content: string) => {
	// ...
});
```